### PR TITLE
Update HTTPS with san.cnf

### PR DESCRIPTION
**Pull request by mccodav1**
Minor change with README updated. This now creates a crt and key from a configuration file rather than a command, and adds the development server to the list of accepted host names.

You will need to use the included command to generate a new .crt and .key file, due to upcoming change in frontend.

### CODE REVIEW

File: san.cnf [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[req]
+default_bits = 4096
+prompt = no
+default_md = sha256
+distinguished_name = dn
+req_extensions = req_ext
+
+[dn]
+C=UnitedStates
```
Comment (atreichl) [inline san.cnf:9]: The country is a two letter code not the whole name, This causes an error on my machine.

**Review by atreichl — CHANGES_REQUESTED** [review]
The subject alt names are not being added to the certificate. Also move the .cnf to deploy would be more convent since then the command would generate the key and cert right where they need to be.  
image

File: san.cnf (hunk 1 above)
Comment (mccodav1) [reply san.cnf]: Whoops.

Everything should be fixed, and the new crt and key are on the dev server (https://postebackend.duckdns.org)

Give it a try on your branch and let me know if it works

**Review by atreichl — APPROVED** [review]
Thank you for move it into deploy. The certificates it generates are valid and can be use to connect to local with host name verification enabled.

**Pull request merged**